nethserver-horde.spec

```diff
@@ -37,3 +37,7 @@ rm -rf %{buildroot}
 %dir %{_nseventsdir}/%{name}-update
 
 %changelog
+* Wed Mar 21 2018 Markus Neuberger <dev@markusneuberger.at> - 0.0.1-1
```

Comment:
Do not alter the `%changelog` section manually during development. Usually changelog entries are added with `release-tag`, when a new RPM is released. See

http://docs.nethserver.org/projects/nethserver-devel/en/v7/building_rpms.html#creating-a-release-tag

Reply:
Thanks for review and infos. It seems to take some time until I fully understand this process. Should I do another PR without the changelog entries?

Reply:
No, you should push a new commit to this PR that fixes the line above. You could even edit the file from GitHub web interface itself ;)

In the future I suggest the following git commands workflow:

- git remote add mrmarkuz <GITURL>
- git fetch origin
- git checkout -b change-proposal origin/master
- edit files
- git commit
- git push mrmarkuz

Then access the origin repo on github and open the PR from your newly pushed `change-proposal` branch.